Integrations/integration-Cisco-umbrella.yml

```diff
@@ -31,495 +31,1834 @@ configuration:
+                    'MoreDataAvailable': nameserver_res['moreDataAvailable'],
+                    'ResultLimit': nameserver_res['limit'],
+                    'Domains': domains_contents
+                })
+                # each nameserver represented by 2 tables
+                # Build Output
+                markdown = markdown + tableToMarkdown('Domains Associated with: ' + nameserver, domains_contents, headers)
+                contents.extend((nameservers_contents, domains_contents))
+
+            context['Umbrella.AssociatedDomains(val.Nameserver && val.Nameserver == obj.Nameserver)'] = {
+                'Nameserver': nameserver,
+                'Data': nameservers_context
             }
-        });
-
-        // context
-        var domainSecurityContext = {};
-        Object.keys(SecurityResultInfo).forEach(function(key) {
-            var contextKey = SecurityResultInfo[key].ContextKey || SecurityResultInfo[key].Name;
-            domainSecurityContext[contextKey] = res[key];
-        });
-        var context = {
-            Domain: {
-                Name: domain,
-                Security: domainSecurityContext
-            }
-        };
 
-        return createTableEntry("Domain Security Info:", table, context);
-    }
+        results.append ({
+            # type
+            'Type': entryTypes['note'],
+            # contents
+            'ContentsFormat': formats['json'],
+            'Contents': contents,
+            # human-readable
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': markdown,
+            # context
+            'EntryContext': context
+        })
+        # Send entries
+        return results
+
+
+    def get_domains_for_nameserver(nameservers, offset, sort, limit):
+        # Build & Send request
+        params = {}
+        # single name server has different api call over multiple name servers
+        if len(nameservers) == 1:
+            endpoint_url = '/whois/nameservers/' + nameservers[0]
+            if sort or limit or offset:
+                params = {
+                    'sortField': sort,
+                    'limit': limit,
+                    'offset': offset
+                }
+        elif len(nameservers) > 1:
+            nameservers_string = ','.join(nameservers)
+            endpoint_url = '/whois/nameservers'
+            if sort or limit or offset:
+                params = {
+                    'nameServerList': nameservers_string,
+                    'sortField': sort,
+                    'limit': limit,
+                    'offset': offset
+                }
+            else:
+                params = {
+                    'nameServerList': nameservers_string,
+                }
 
-    var getDomainTags = function(domain) {
-        var url = baseURL + '/domains/' + encodeURIComponent(domain) + '/latest_tags';
-        var tags = sendRequest(url);
+        res = ht
```

Comment:
Perhaps you should create a CONST list for each command and the use `demisto.command() in LIST`

Reply:
Did similar thing in minemeld back then, @anara123 is not a fan of it.